README.md

```diff
@@ -0,0 +1,58 @@
+# nagios-plugins-bdii
+
+This nagios probe is used to monitor BDII.
+It provides check_bdii_entries support for Nagios.
+
+BDII documentation is available here: https://gridinfo-documentation.readthedocs.io/
+
+## Building packages
```

Comment:
mention that they are built and delivered by Travis too and can be found in the releases part of the repo.

Reply:
They aren't for the time being in fact, no travis yet for thi repo :)
Please open a ticket to track this.